### DISCUSSION TITLE
HorizontalLayout.FullWidth has no labels when there are 2 entries only

### DISCUSSION THREAD

**Discussion by zoltish** [patrykandpatrick/vico#437] [Ideas]
Could be related to #394 

While I understand _why_ this happens, I would love to see a way to show the first/last label in scenarios like this! From what I can tell, my only other option is to make the chart segmented - but that kind of defeats the purpose, and results in bad UX. 

<img width="418" alt="image" src="https://github.com/patrykandpatrick/vico/assets/4264531/96260300-6384-4577-85a8-327f5527f129">

**Comment by patrickmichalik** [comment]
Hello! Thanks for the suggestion. In addition to using `HorizontalLayout.Segmented`, you can use `HorizontalLayout.FullWidth`’s padding feature to make room for the labels. Vico 1.13.0 Alpha 9, which we plan on releasing within a couple of hours, will introduce a feature that automatically adds the required amount of padding, as requested in #431.

**Comment by patrickmichalik** [comment]
I wanted to follow up and let you know that [Vico 1.13.0 Alpha 9](https://github.com/patrykandpatrick/vico/releases/tag/v1.13.0-alpha.9) is now available. It was released on schedule, around two hours after I posted my previous comment. If you have any questions, please feel free to let me know. Cheers!

**Reply by zoltish** [reply]
Its been great, thank you! 👏🏾 

I thought Id follow up your follow-up with something that came to mind: 

Would it be feasible to render the first and last label with a different alignment (START; CENTER; CENTER ... ; END) so that the chart fills as much of the chart area as possible? 

Currently my label length dictates where the chart starts and ends ("October", "November"); it doesnt look terribly bad but I think it would look better with this tweak.

**Reply by patrickmichalik** [reply]
I’m glad to hear automatic label padding works as expected. We’ll be looking into adding label alignment customization—please see #428.
